zone/hate_list.cpp

```diff
@@ -745,7 +745,7 @@ void HateList::SpellCast(Mob *caster, uint32 spell_id, float range, Mob* ae_cent
 	//crashes when people kick the bucket in the middle of this call
 	//that invalidates our iterator but there's no way to know sadly
 	//So keep a list of entity ids and look up after
-	std::list<uint32> id_list;
+	std::list<uint64> id_list;
 	range = range * range;
```

Comment:
Intentional change here?

Reply:
Nope, won't hurt anything but take 4 more bytes of register memory per entry. I'll probably bring this one back down as I work through this more in a bit.

Reply:
No worries, just making note since it's spell ID and stuff that had changed.